src/ddmd/attrib.d

```diff
@@ -165,6 +165,9 @@ extern (C++) abstract class AttribDeclaration : Dsymbol
 
     override void semantic(Scope* sc)
     {
+        if (semanticRun != PASSinit)
```

Comment:
No. But such possible odd behaviors are why I've been doing these incrementally. If they fail, it is easier to track things down.